libs/vm-modules/src/ml/model/model_estimator.cpp

```diff
@@ -107,7 +110,11 @@ ChargeAmount ModelEstimator::LayerAddDenseActivation(Ptr<fetch::vm::String> cons
 {
   ChargeAmount estimate = LayerAddDense(layer, inputs, hidden_nodes);
 
-  FETCH_UNUSED(activation);  // only relu is valid
+  if (activation->string() != "relu")
+  {
+    return MaximumCharge("Only relu activations are supported");
```

Comment:
@iryna-melnyk, do we have any changes regarding other activations in VMModel?

Reply:
I guess not yet